### PR TITLE
Feature propeller polars fix

### PR DESCRIPTION
**Pull request by rachealerhard**
# Pull Request Checklist
 * Merge in the latest Develop branch changes to your branch
 * Remove .pyc files from your repository:
   * Linux/Mac
     * find . -name *.pyc -delete
   * Windows
     * del /S *.pyc
 * Run automatic regression and makes sure everything is passing
 * Did you make a new regression test that covers your new code?
 * Did you update your docstrings?
 * Did you update your headers to include your name and date?
 * Do a final compare with the Develop branch to be sure what you're changing

### CODE REVIEW

File: trunk/SUAVE/Methods/Geometry/Two_Dimensional/Cross_Section/Airfoil/compute_airfoil_polars.py [hunk 1]
```diff
@@ -149,7 +148,30 @@ def compute_airfoil_polars(a_geo,a_polar):
             # Pack this loop
             CL[i,j,:] = CL_ij
             CD[i,j,:] = CD_ij
-           
+            
+            if use_pre_stall_data == True:
```
Comment (planes) [inline trunk/SUAVE/Methods/Geometry/Two_Dimensional/Cross_Section/Airfoil/compute_airfoil_polars.py:152]: I think this should be a separate function. The reason is that AERODAS for aerodynamics and for propellers can both use this code without duplication

File: trunk/SUAVE/Methods/Geometry/Two_Dimensional/Cross_Section/Airfoil/compute_airfoil_polars.py [hunk 2]
```diff
@@ -149,7 +148,30 @@ def compute_airfoil_polars(a_geo,a_polar):
...
+                # Coefficients in pre-stall regime taken from experimental data:
+                aoa_locs = (AoA_sweep_deg>=airfoil_aoa[0]) * (AoA_sweep_deg<=airfoil_aoa[-1])
+                aoa_in_data = AoA_sweep_deg[aoa_locs]
+                
+                cl_exp = np.zeros_like(aoa_in_data)
+                cd_exp = np.zeros_like(aoa_in_data)
+                for a in range(len(aoa_in_data)):
```
Comment (planes) [inline trunk/SUAVE/Methods/Geometry/Two_Dimensional/Cross_Section/Airfoil/compute_airfoil_polars.py:159]: This loop  is vectorizable. Let me know if you have questions about that.

File: trunk/SUAVE/Methods/Geometry/Two_Dimensional/Cross_Section/Airfoil/compute_airfoil_polars.py [hunk 3]
```diff
@@ -160,6 +182,10 @@ def compute_airfoil_polars(a_geo,a_polar):
     airfoil_data.lift_coefficient_surrogates   = CL_surs
     airfoil_data.drag_coefficient_surrogates   = CD_surs 
     
+    airfoil_data.cl_airfoiltools  = airfoil_polar_data.lift_coefficients
```
Comment (planes) [inline trunk/SUAVE/Methods/Geometry/Two_Dimensional/Cross_Section/Airfoil/compute_airfoil_polars.py:185]: let's not name this airfoiltools

File: trunk/SUAVE/Methods/Geometry/Two_Dimensional/Cross_Section/Airfoil/compute_airfoil_polars.py (hunk 2 above)
Comment (rachealerhard) [reply trunk/SUAVE/Methods/Geometry/Two_Dimensional/Cross_Section/Airfoil/compute_airfoil_polars.py:159]: done

**Review by planes — APPROVED** [review]

**Pull request merged**